app/views/admin/loans/questionnaires/_questionnaire_toc.html.slim

```diff
@@ -1,6 +1,6 @@
-  ol
+  ul
```

Comment:
Currently the Toc numbering is done in Css. It does not use the same method as the body, so the numbers would not line up. Chatted to @linesarefuzzy about this one, and because the numbering is going to be refactored soon, we decided to just disable it for now on the Toc.